_raw/locales/ru/messages.json

```diff
@@ -0,0 +1,1654 @@
+        "customizedButton": "настроено",
+        "blockedButton": "заблокировано"
+      },
+      "assets": {
+        "usdValue": "ОЦЕНКА В USD",
+        "amount": "КОЛ-ВО",
+        "portfolio": {
+          "nftTips": "Рассчитано на основе нижней цены, определенной этим протоколом.",
+          "fractionTips": "Рассчитано на основе цены связанного токена ERC20."
+        },
+        "tokenButton": {
+          "subTitle": "Токен из этого списка не будет включен в общий баланс."
+        },
+        "table": {
+          "assetAmount": "Актив / Количество",
+          "price": "Цена",  
+          "useValue": "Стоимость в USD",
+          "healthRate": "Коэффициент надежности",
+          "debtRatio": "Коэффициент задолженности",
+          "unlockAt": "Разблокировать в",
+          "lentAgainst": "ВЫДАНО В ЗАЛОГ",
+          "type": "Тип",
+          "strikePrice": "Цена исполнения",
+          "exerciseEnd": "Окончание исполнения",
+          "tradePair": "Торговая пара",  
+          "side": "Сторона",
+          "leverage": "Кредитное плечо", 
+          "PL": "P&L",
+          "unsupportedPoolType": "Неподдерживаемый тип пула",
+          "claimable": "Доступно к выводу",
+          "endAt": "Заканчивается в",
+          "dailyUnlock": "Ежедневная разблокировка",
+          "pool": "ПУЛ",
+          "token": "Токен",
+          "balanceValue": "Баланс / Стоимость",
+          "percent": "Процент",
+          "summaryTips": "Стоимость активов, разделенная на общую чистую стоимость",
+          "summaryDescription": "Все активы в протоколах (например, токены LP) преобразуются в базовые активы для статистических расчетов",
+          "noMatch": "Нет совпадений",
+          "lowValueDescription": "Активы с низкой стоимостью будут показаны здесь", 
+          "lowValueAssets": "{{count}} активов с низкой стоимостью"
+        },
+        "noAssets": "Нет активов",
+        "blockLinkText": "Искать адрес, чтобы заблокировать токен",
+        "blockDescription": "Заблокированные вами токены будут показаны здесь",
+        "unfoldChain": "Развернуть 1 сеть",  
+        "unfoldChainPlural": "Развернуть {{moreLen}} сетей",
+        "customLinkText": "Искать адрес, чтобы добавить пользовательский токен",
+        "customDescription": "Добавленные вами пользовательские токены будут показаны здесь",
+        "comingSoon": "Скоро...",
+        "searchPlaceholder": "Токены"
+      },
+      "hd": {
+        "howToConnectLedger": "Как подключить Ledger",
+        "userRejectedTheRequest": "Пользователь отклонил запрос.",  
+        "ledger": {
+          "doc1": "Подключите один Ledger",
+          "doc2": "Введите пин-код для разблокировки",
+          "doc3": "Откройте приложение Ethereum",
+          "reconnect": "Если это не работает, пожалуйста, попробуйте <1>подключиться сначала</1>."
+        },
+        "howToSwitch": "Как переключить"
+      }
+    },
+    "nft": {
+      "floorPrice": "/ Floor Price:",
```

Comment:
Is the `/` typo?

Reply:
Yeah, looks like. Original file has it too.